src/app/modules/work-time/work-time.component.html

```diff
@@ -4,13 +4,13 @@
         <tbody>
         <tr>
             <th title="Time period">Period</th>
-            <th class="week-name-table" title="Sunday">Su</th>
-            <th class="week-name-table" title="Monday">Mo</th>
-            <th class="week-name-table" title="Tuesday">Tu</th>
-            <th class="week-name-table" title="Wednesday">We</th>
-            <th class="week-name-table" title="Thursday">Th</th>
-            <th class="week-name-table" title="Friday">Fr</th>
-            <th class="week-name-table" title="Saturday">Sa</th>
+            <th class="week-name-table" title="Sunday">{{ getDayFormat('Sunday') }}</th>
```

Comment:
You could do it better and faster, if you will use a object with presaving names:
Example:
```
            <th class="week-name-table" title="Saturday">{{longFormat['sa'] | uppercase}}</th>
```